docs/authentication/authentication.rst

```diff
@@ -48,6 +48,18 @@ The IDP may ask for the following settings:
 
 #. Custom attributes - Mautic requires three custom attributes in the IDP responses for the User Email, first name and last name. Username is also supported but is optional. Configure the attribute names used by the IDP in Mautic's Configuration > User/Authentication Settings under the 'Enter the names of the attributes the configured IDP uses for the following Mautic User fields' section.
 
+Azure Specific Configuration
+============================
+
+For Azure you need to make sure to put the complete XML schema with path instead of just Attribute name. So within Attributes do not put them like "emailaddress" or "surname". See below example for 3 required fields"
```

Comment:
A few fixes suggested by Vale for grammar and I've dropped the quoted text in an inline code block because they aren't speech, so don't belong in speech marks.

```suggestion
For Azure you need to enter the complete XML schema including the path, rather than just the attribute name. For example, within ``Attributes`` don't enter the values as ``emailaddress`` or ``surname``.  Instead, see the following example:
```